schema/schema.cc

```diff
@@ -943,6 +943,41 @@ std::ostream& schema::describe(replica::database& db, std::ostream& os, bool wit
     if (is_compact_table()) {
         os << "COMPACT STORAGE\n    AND ";
     }
+    schema_options(db, os);
+    os << ";\n";
+
+    if (with_internals) {
+        for (auto& cdef : dropped_columns()) {
+            os << "\nALTER TABLE " << cql3::util::maybe_quote(ks_name()) << "." << cql3::util::maybe_quote(cf_name())
+               << " DROP " << cql3::util::maybe_quote(cdef.first) << " USING TIMESTAMP " << cdef.second.timestamp << ";";
+
+            auto column = get_column_definition(to_bytes(cdef.first));
+            if (column) {
+                os << "\nALTER TABLE " << cql3::util::maybe_quote(ks_name()) << "." << cql3::util::maybe_quote(cf_name())
+                   << " ADD ";
+                column_definition_as_cql_key(os, *column);
+                os << ";";
+            }
+        }
+    }
+
+    return os;
+}
+
+sstring schema::alter_table_with_options(replica::database& db) const {
```

Comment:
Nit: I think the first commit would be better split into two commits:

- First, introduce `schema_options` (code movement)
- Then introduce `alter_table_with_options` (new functionality)

It would be a bit easier for the reviewers to verify the first step if the commit were split like this.

Reply:
Besides, the commit message doesn't say anything about the `schema_options` method...